scrnaseq.Rmd

```diff
@@ -861,12 +877,14 @@ if (param$norm == "RNA") {
   # Note: It is not guaranteed that all genes are successfully normalised with SCTransform. 
   #   Consequently, some genes might be missing from the SCT assay. 
   #   See: https://github.com/ChristophH/sctransform/issues/27
+  # Note: The performance of SCTransform can be improved by using 'glmGamPoi' instead of 'poisson' as method for initial parameter estimation.
   sc = purrr::map(list_names(sc), function(n) { 
     SCTransform(sc[[n]], 
                 vars.to.regress=param$vars_to_regress, 
                 min_cells=param$feature_filter[[n]][["min_cells"]], 
                 verbose=FALSE, 
-                return.only.var.genes=!(param$integrate_samples[["method"]]=="single")) 
+                return.only.var.genes=!(param$integrate_samples[["method"]]=="single"),
+                method=ifelse(packages_installed("glmGamPoi"), "glmGamPoi", "poisson")) 
```

Comment:
What is this about? Why not install the package if it is not installed?

Reply:
The package is only available for R 4.0.0. Therefore I would not like to enforce this for now.